zee/src/editor/mod.rs

```diff
@@ -217,6 +217,15 @@ impl Component for Editor {
                 link.send(Message::SplitWindow(FlexDirection::Row));
             }
         }
+
+        let mut theme_index = 0;
+        for (i, (_, name)) in THEMES.iter().enumerate() {
+            if *name == properties.settings.theme_name {
+                theme_index = i;
+                break;
+            }
+        }
```

Comment:
This would be clearer with iterator combinators, in particular using [position](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.position) -- additionally, it'd avoid the use of `mut` and make the default case when the theme name cannot be found more explicit (e.g. `.unwrap_or(0)`)

Reply:
Great suggestion, I wasn't aware of that API. Updated as suggested.